geometry-validation/vgapp/vgapp.cc

```diff
@@ -72,7 +77,9 @@ std::ostream& operator<<(std::ostream& os, const vgapp::VolumeMap& map)
     {
         os << "| " << std::left << std::setw(width_ids) << key.first << " | "
            << std::setw(width_material) << key.second.material_name << " | "
-           << std::setw(width_volume) << key.second.volume_name << " |"
+           << std::setw(width_volume) << key.second.volume_name << " | "
```

Comment:
Suggestion to add more columns so renders nicely when opening the `.md` file in a viewer. Here is how the whole function would look like:

```cpp
std::ostream& operator<<(std::ostream& os, const vgapp::VolumeMap& map)
{
    size_t width_ids      = 7;
    size_t width_volume   = 0;
    size_t width_material = 0;
    size_t width_placed   = 11;
    size_t width_copy     = 12;

    for (const auto& it : map)
    {
        width_volume = std::max(width_volume, it.second.volume_name.size());
        width_material
            = std::max(width_material, it.second.material_name.size());
    }

    // Titles line
    os << std::endl;
    os << "| " << std::left << std::setw(width_ids) << "Vol ID"
       << " | " << std::setw(width_material) << "Material"
       << " | " << std::setw(width_volume) << "Volume"
       << " | " << std::setw(width_placed) << "Num placed"
       << " | " << std::setw(width_copy) << "Min copy num"
       << " | " << std::setw(width_copy) << "Max copy num |" << std::endl;

    // Dashed line
    os << "| ";
    for (int i = 0; i < width_ids; i++)
        os << "-";
    os << " | ";
    for (int i = 0; i < width_material; i++)
        os << "-";
    os << " | ";
    for (int i = 0; i < width_volume; i++)
        os << "-";
    os << " | ";
    for (int i = 0; i < width_placed; i++)
        os << "-";
    os << " | ";
    for (int i = 0; i < width_copy; i++)
        os << "-";
    os << " | ";
    for (int i = 0; i < width_copy; i++)
        os << "-";
    os << " |";
    os << std::endl;

    // Table content
    for (const auto key : map)
    {
        os << "| " << std::left << std::setw(width_ids) << key.first << " | "
           << std::setw(width_material) << key.second.material_name << " | "
           << std::setw(width_volume) << key.second.volume_name << " | "
           << std::setw(width_placed) << key.second.numPlaced << " | "
           << std::setw(width_copy) << key.second.copyNumMin << " | "
           << std::setw(width_copy) << key.second.copyNumMax << " |"
           << std::endl;
    }

    return os;
}
```